src/fsharp/FSStrings.resx

```diff
@@ -970,7 +970,7 @@
     <value>This rule will never be matched</value>
   </data>
   <data name="ValNotMutable" xml:space="preserve">
-    <value>This value is not mutable</value>
+    <value>Values are immutable by default. If you intend to mutate this value, declare it using the mutable keyword, e.g. 'let mutable {0} = expression'.</value>
```

Comment:
Thanks for updating. Looking at this one last time, I wonder why not keep the original wording for the first part?  Somehow it feels better now I look at it yet again 

    This value is not mutable. If you intend to mutate this value, declare it using the mutable keyword, e.g. 'let mutable {0} = expression'

c.f.

    Values are immutable by default. If you intend to mutate this value, declare it using the mutable keyword, e.g. 'let mutable {0} = expression' 

Somehow the first describes the reality of the situation without needing to mention 'by default'?